sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -458,15 +461,24 @@ export class AssessmentSequentialProcessingService {
       .orderBy("assessment.assessmentDate")
       .limit(1)
       .getQuery();
-    // Sub query to determined if the assessment has at least one non-declined COE (Required or Completed).
-    // If all the COEs from the assessment are declined some user action will be needed in the application
-    // and this application will not be considered for sequential processing.
-    const existsValidCOE = entityManager
+    // Sub query to determine if the application has at least one payable disbursement.
+    // If all the disbursements from the application are NOT payable then the application will not be considered for sequential processing.
+    const existsValidDisbursement = entityManager
       .getRepository(DisbursementSchedule)
       .createQueryBuilder("disbursementSchedule")
       .select("1")
-      .where("disbursementSchedule.studentAssessment.id = currentAssessment.id")
-      .andWhere("disbursementSchedule.coeStatus != :declinedCOEStatus")
+      .innerJoin(
+        "disbursementSchedule.studentAssessment",
+        "disbursementAssessment",
+      )
+      .innerJoin(
+        "disbursementAssessment.application",
+        "disbursementApplication",
+      )
+      .where("disbursementApplication.id = application.id")
```

Comment:
I do not understand why this relation was needed explicitly. This one seems like a regular Typeorm entity relation that the entity model should resolve.